desktopexporter/internal/store/util.go

```diff
@@ -0,0 +1,243 @@
+package store
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/CtrlSpice/otel-desktop-viewer/desktopexporter/internal/telemetry"
+)
+
+// Helper function to convert Events to DuckDB list of STRUCT string format
+func eventToString(events []telemetry.EventData) string {
```

Comment:
I know these are meant to be temporary, and we can validate their functioning by running the code, but tests for these would be pretty simple to write and help if we ever need to modify these. They'd also serve as documentation.